envs_manager/api.py

```diff
@@ -49,6 +53,18 @@ def __init__(self, environment_path, external_executable=None):
         self.executable_variant = None
         assert self.validate(), f"{self.ID} backend unavailable!"
 
+    def _get_package_info(self, package_name, channel=None):
```

Comment:
Please add the explanation you provided below to the docstring of this method so future maintainers understand why it's defined like this.